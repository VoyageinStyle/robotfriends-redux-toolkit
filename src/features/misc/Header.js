import React from "react";

const Header = React.memo(() => {
  return <h1>RobotFriends</h1>;
})

export default Header