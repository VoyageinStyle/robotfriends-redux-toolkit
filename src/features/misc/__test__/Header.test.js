import React from "react";
import { render, screen } from "@testing-library/react";
import  "@testing-library/jest-dom"
import Header from "../Header";

it("displays a header", () => {
  render(<Header />);
  expect(screen.getByText("RobotFriends")).toBeInTheDocument();
});
it("never changes", () => {
  render(<Header />);
  expect(screen.getByText("RobotFriends")).toMatchSnapshot();
});