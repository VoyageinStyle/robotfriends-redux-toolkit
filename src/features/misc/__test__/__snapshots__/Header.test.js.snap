// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`never changes 1`] = `
<h1>
  RobotFriends
</h1>
`;
